### DISCUSSION TITLE
Quiz: 1.4 Part 5 -  Latex not rendering correctly

### DISCUSSION THREAD

**Discussion by ssuzana** [Lightning-AI/dl-fundamentals#74] [Lecture videos and quizzes] [answered]
Hi Sebastian,

I noticed that in one of the questions in Quiz: 1.4 Part 5 Latex is not rendering correctly. I'm using Chrome.

Here is a picture of how it looks like:

<img width="796" alt="Screenshot 2023-07-13 at 4 04 54 PM" src="https://github.com/Lightning-AI/dl-fundamentals/assets/74042651/65c84f47-282b-472d-9e69-9883c6e1ce27">

**Comment by rasbt** [comment, marked as answer]
Thanks for sending this in! I believe I fixed it now. There's some caching on the server so it may take an hour for it to take effect. I'll post the URL here so that I can double-check later. https://lightning.ai/courses/deep-learning-fundamentals/unit-1/1-5-computing-environment/

**Comment by ssuzana** [comment]
No problem, it looks fixed on my end:
https://lightning.ai/courses/deep-learning-fundamentals/unit-1/ml-classifier/
<img width="834" alt="Screenshot 2023-07-15 at 10 03 00 AM" src="https://github.com/Lightning-AI/dl-fundamentals/assets/74042651/428ca2de-1681-42f5-94ee-c0b778ac1b8e">

**Reply by rasbt** [reply]
Awesome, glad to hear!!
